### ISSUE TITLE
Guards walk at villager speed until targets enemy mob

### ISSUE THREAD

**Issue by jengendler** [seymourimadeit/guardvillager-suggestions#8]
It's sometimes difficult to catch up to guards (to change their gear) because they constantly walk faster than villagers. It would be useful to have a config feature to make them walk at villager speed until they target an enemy mob.

**Comment by ghost** [comment]
I agree, and it makes no sense why they would run so fast when patrolling. It doesn't look too good.
